packages/components-library/src/components/filters/FilterCard.vue

```diff
@@ -0,0 +1,200 @@
+    <b-collapse
+      :id="name"
+      v-model="isOpen"
+    >
+      <b-card-body>
+        <b-form-group :description="description">
+          <slot />
+        </b-form-group>
+      </b-card-body>
+    </b-collapse>
+  </b-card>
+</template>
+
+<script>
+import Vue from 'vue'
+
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faCaretRight, faTimes } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+library.add(faCaretRight, faTimes)
+
+export default Vue.extend({
+  name: 'FilterCard',
+  components: { FontAwesomeIcon },
+  props: {
+    /**
+     * Unique filter identifier
+     */
+    name: {
+      type: String,
+      required: true
+    },
+    /**
+     * Human readable label descibing the filter
+     */
+    label: {
+      type: String,
+      required: false,
+      default: () => ''
+    },
+    /**
+     * Indicates if filter is shown in collapsed state or not
+     */
+    collapsed: {
+      type: Boolean,
+      required: false,
+      default: () => true
+    },
+    /**
+     * Indicates if the user can collapse the filter
+     */
+    collapsable: {
+      type: Boolean,
+      required: false,
+      default: () => true
+    },
+    /**
+     * Human readable text with aditional description of the filter
+     */
+    description: {
+      type: String,
+      required: false,
+      default: () => ''
+    },
+    /**
+     * Whether or not the user can remove the filter or not
+     */
+    canRemove: {
+      type: Boolean,
+      required: false,
+      default: () => false
+    }
+  },
+  data () {
+    return {
+      isOpen: this.collapsable ? !this.collapsed : true
+    }
+  },
+  computed: {
+    iconStyle () {
+      return {
+        transform: `rotate(${this.isOpen ? 90 : 0}deg)`,
+        transition: 'transform 0.2s'
+      }
+    }
+  },
+  methods: {
+    removeFilter () {
+      this.$emit('removeFilter', this.name)
+    },
+    toggleState () {
+      if (this.collapsable) {
+        this.isOpen = !this.isOpen
+        return this.isOpen
+      }
+      return false
+    }
+  }
+})
+</script>
+
+<style scoped>
+  .card-header.collapsable {
+    cursor: pointer;
+  }
+  .form-group {
+    margin-bottom:0;
+  }
+  .title{
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+  .remove-button{
+    transition: opacity 0.2s, color 0.2s;
+    opacity: 0;
+    height: inherit;
+    width: 1.5em;
+    text-align: center;
+    display: inline-block;
+    position: absolute;
+    right: 10px;
+    cursor: pointer;
+  }
+  .remove-button:hover{
+    color: var(--danger);
+  }
+  .filter-card:hover .remove-button{
+    opacity: 1;
+  }
+  .sortable-ghost{
+    border-style: dashed;
+  }
+  .sortable-ghost > div{
+    opacity: 0.2;
+  }
+</style>
+
+<docs>
+Card containing filter
+### Usage
+```jsx
+const onRemoveFilter = () => alert('remove me please')
+
+const options = () => Promise.resolve(
+ [
```

Comment:
It seems a bit confusing that FilterCard is in the Filter category, while it's just a container for a filter. Why not create a new category "Containers" or something the like.

Reply:
done see: https://github.com/molgenis/molgenis-frontend/pull/391/commits/565175b438ad4443b97a011d196dd153ede3dd6b